com.unity.template-hd/Assets/SampleSceneAssets/Tutorials/Settings/TutorialCallbacks.cs

```diff
@@ -0,0 +1,61 @@
+﻿using UnityEngine;
```

Comment:
I dont get the purpose of this script. Callbacks are never called anywhere. There is a ScriptableObject created from it but its GUID is nowhere which means it is used in no scene/prefab/assets. Is it some dead code to remove ?

Reply:
@mikaisomaa, is this TutorialCallbacks file absolutely necessary?

I did re-generate it, because upgrading from the preview version of the IET package to the new one created errors on this script. However, it doesn't indeed seem to be necessary at all.

Is it safe to remove?

Reply:
By the way, the scroll bar is a bug that the IET team will investigate.